client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -198,7 +246,9 @@ tie.factory('PythonCodePreprocessorService', [
           buggyOutputTests, performanceTests) {
         return [
           SYSTEM_CODE,
-          this._wrapCodeIntoClass(studentCode, CLASS_NAME_STUDENT_CODE),
+          this._wrapCodeIntoClass(
```

Comment:
Can we reframe this as this._wrapCodeIntoClass(..., ..., true/false) where the third arg indicates whether to process internal function refs or not? I think that would be cleaner and more general. (It's still fine to pull out a separate function to do the actual "add function prefix" processing, though, but that would be called from _wrapCodeIntoClass().)